src/main/java/ben/commands/MarkCommand.java

```diff
@@ -0,0 +1,49 @@
+package ben.commands;
+
+import ben.exceptions.BenException;
+import ben.storage.Storage;
+import ben.tasks.TaskList;
+import ben.ui.Ui;
+
+/**
+ * Represents a command to mark a task as done in the task list.
+ */
+public class MarkCommand extends Command {
+    private final int index;
```

Comment:
I like that your class variables are never declared public which follows the coding standard